include/llzk/Transforms/LLZKTransformationPasses.td

```diff
@@ -70,4 +70,23 @@ def PolyLoweringPass : LLZKPass<"llzk-poly-lowering-pass"> {
   ];
 }
 
+def InlineStructsPass : LLZKPass<"llzk-inline-structs"> {
+  let summary = "Inlines nested structs (i.e. subcomponents).";
```

Comment:
Nit: "i.e." is followed by a comma according to American style, but apparently not followed by a comma in British standards. In honor of our upcoming celebration on July 4th, I feel like adding commas here and below would be the patriotic thing to do.

Reply:
😆🇺🇸